src/__tests__/ProgressCircle-test.jsx

```diff
@@ -55,9 +55,5 @@ describe('ProgressCircle', function() {
       );
       let comp = TestUtils.findRenderedComponentWithType(reactTree, ProgressCircle);
       expect(comp.state.pathLength).toEqual(circumference);
-
-      let path = TestUtils.findRenderedDOMComponentWithClass(reactTree, 'progress-circle--path');
-      expect(parseFloat(path._style.strokeDashoffset)).toBeCloseTo(dashOffset, 6);
```

Comment:
This test was throwing an 'undefined' error. Probably a syntax change or something that sort with the new version of Jest or React. I will dedicate a sprint or half in future to add more tests to all of our components.